src/pages/api/thirdparty/dataSources/thirdPartyKeystone.ts

```diff
@@ -0,0 +1,36 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+import type { KeyValueCache } from '@apollo/utils.keyvaluecache'
+import { DateTime } from 'luxon'
+import { print } from 'graphql'
+import { GET_PUBLIC_ARTICLES } from '../operations/getPublicArticles'
+
+export const ARTICLE_PUBLIC_TAGS = {
+  CNOTE: 'C-Note',
+}
+
+class ThirdPartyKeystoneAPI extends RESTDataSource {
+  override baseURL = process.env.KEYSTONE_URL
+
+  constructor(options: { cache: KeyValueCache }) {
+    super(options)
+  }
+
+  async getPublicArticles(publishedDate: DateTime, tag?: string) {
```

Comment:
Question: I'm missing how this works when you don't pass in the tag. Does apollo client return all data across all the enum values? We only have one so it isn't immediately clear to me.